### PR TITLE
chore: set account details for honeycomb metrics

### PR DESCRIPTION
**Pull request by dmurray-lacework**
Set additional properties for honeycomb metrics

### CODE REVIEW

File: api/metrics.go [hunk 1]
```diff
@@ -18,16 +18,41 @@
...
 	client *Client
 }
 
 func (svc *MetricsService) Send(event Honeyvent) (response HoneyEventResponse, err error) {
+	event.setAccountDetails(*svc.client)
 	err = svc.client.RequestEncoderDecoder("POST", apiV2HoneyMetrics, event, &response)
 	return
 }
 
+func NewHoneyvent(version, feature, dataset string) Honeyvent {
```
Comment (ipcrm) [inline api/metrics.go:34]: This doesn't seem to be used?

**Review by ipcrm — COMMENTED** [review]
Question on the `NewHoneyvent` function

File: api/metrics.go (hunk 1 above)
Comment (dmurray-lacework) [reply api/metrics.go:34]: At the moment it's not used in the go-sdk and cli. But it is used by terraform provider.

Comment (dmurray-lacework) [reply api/metrics.go:34]: See this PR for an example -> https://github.com/lacework/terraform-provider-lacework/pull/611/files#diff-145f8ae7d249239fe57b8eb4e7bd92129c5606ae1a30a2bd1ee9615ff7b26093R47

Comment (ipcrm) [reply api/metrics.go:34]: ty!

**Review by ipcrm — APPROVED** [review]

**Pull request merged**